apps/sms/js/navigation.js

```diff
@@ -42,17 +42,17 @@ var Navigation = {
 
   panels: {
     'thread': {
-      behaviour: 'ThreadUI',
+      behaviour: 'ConversationView',
       wrapperPosition: 'left',
       container: 'thread-messages'
     },
     'thread-list': {
-      behaviour: 'ThreadListUI',
+      behaviour: 'InboxView',
       wrapperPosition: 'right',
       container: 'thread-list'
```

Comment:
I didn't change names of the views in Navigation - I think it's a separate task.